tests/TestCase/Network/ResponseTest.php

```diff
@@ -2778,6 +2778,52 @@ public function testWithBody()
     }
 
     /**
+     * Test with string body.
+     *
+     * @return void
+     */
+    public function testWithStringBody()
+    {
+        $response = new Response();
+        $response->withStringBody('Foo');
+        $body = $response->getBody();
+        $this->assertSame('Foo', (string)$body);
+
+        $response = new Response();
+        $response->withStringBody('');
+        $body = $response->getBody();
+        $this->assertSame('', (string)$body);
+
+        $response = new Response();
+        $response->withStringBody(null);
+        $body = $response->getBody();
+        $this->assertSame('', (string)$body);
+
+        $response = new Response();
+        $response->withStringBody(1337);
+        $body = $response->getBody();
+        $this->assertSame('1337', (string)$body);
+    }
```

Comment:
also you need to ensure you created new obj with `withStringBody()` but i not see this assert